src/librustc/hir/mod.rs

```diff
@@ -1118,7 +1118,7 @@ impl BinOpKind {
         }
     }
 
-    /// Returns `true` if the binary operator takes its arguments by value.
+    /// Returns `true` if the binary operator takes its parameters by value.
```

Comment:
"arguments" is correct because it's like a call, not like a function definition.